app/controllers/notifications_controller.rb

```diff
@@ -0,0 +1,27 @@
+class NotificationsController < ApplicationController
+  def index; end
+
+  def create
+    if params[:consent].nil?
```

Comment:
`params.require(:consent)` (as per line 12) ?

Reply:
The `require` seemed to blow up when it wasn't passed in. (I.e the user didn't select an option)